deploy/craned/deployment.yaml

```diff
@@ -150,6 +150,17 @@ metadata:
   name: nginx-conf
   namespace: crane-system
 data:
+  common_params: |
```

Comment:
How to make it take effect when you move to here?

Reply:
include conf.d/common_params;